pandas/core/generic.py

```diff
@@ -9355,15 +9355,23 @@ def align(
             - pad / ffill: propagate last valid observation forward to next valid.
             - backfill / bfill: use NEXT valid observation to fill gap.
 
+            .. deprecated:: 2.1
+
         limit : int, default None
```

Comment:
Should we change the defaults to lib.no_default? (for all keywords)

Reply:
i guess thatd be harmless, OK

Reply:
good call here, we had a few places where we explicitly passed method=None that werent caught in the previous version.

want to weigh in on broadcast_axis in #51856?